src/mapped-types.ts

```diff
@@ -614,3 +614,26 @@ export type ValuesType<
   : T extends object
   ? T[keyof T]
   : never;
+
+/**
+ * Required
+ * @desc From `T` make a set of properties by key `K` become required
+ * @example
+ *    type Props = {
+ *      name?: string;
+ *      age?: number;
+ *      visible?: boolean;
+ *    };
+ *
+ *    // Expect: { name: string; age: number; visible: boolean; }
+ *    type Props = Required<Props>;
+ *
+ *    // Expect: { name?: string; age: number; visible: boolean; }
+ *    type Props = Required<Props, 'age' | 'visible'>;
+ */
+type AugmentedRequired<T extends object, K extends keyof T = keyof T> = Omit<
+  T,
+  K
+> &
+  Required<Pick<T, K>>;
+export { AugmentedRequired as Required };
```

Comment:
Could you please use normal export on AugmentedRequired here in this module for clearer internal references, and move named export to the API exports in the main index.ts?